proto/dydxprotocol/perpetuals/query.proto

```diff
@@ -59,6 +65,14 @@ message QueryAllPerpetualsResponse {
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
 
+// Queries a list of LiquidityTier items.
+message QueryAllLiquidityTiersRequest {}
```

Comment:
note that I didn't think this needs to be paginated - let me know if anyone has a preference for this

Reply:
actually on a second thought - let me just add pagination to be consistent with other queries and to be more future proof

Reply:
agreed